src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -64,22 +65,52 @@ public sealed class PolicyContract : NativeContract
         /// </summary>
         public const uint MaxStoragePrice = 10000000;
 
+        /// <summary>
+        /// The minimum block generation time that the committee can set in milliseconds.
+        /// </summary>
+        public const uint MinBlockGenTime = 100;
```

Comment:
```suggestion
        public const uint MinBlockGenTime = 0;
```

It was like this in the last version of this PR, but the latest commit changed it.

Reply:
Why the min is 0 ? is not it 100 in our last meeting?

Reply:
No. As mentioned in https://github.com/neo-project/neo/pull/3622#discussion_r1984840756, 100 is not enough for us.